dbt/adapters/spark_cde/cdeapisession.py

```diff
@@ -320,10 +320,11 @@ class CDEApiConnection:
         "failed": "failed",
     }
 
-    def __init__(self, base_api_url, access_token, api_header) -> None:
+    def __init__(self, base_api_url, access_token, api_header, session_params) -> None:
```

Comment:
Are session parameters mandatory now?

Reply:
Session parameters are not mandatory. Default value is set where parameters are read - in connections.py